document/js-api/index.bs

```diff
@@ -748,7 +751,11 @@ This slot holds a [=function address=] relative to the [=surrounding agent=]'s [
     1. Set the [=surrounding agent=]'s [=associated store=] to |store|.
     1. If |ret| is [=error=], throw an exception. This exception should be a WebAssembly {{RuntimeError}} exception, unless otherwise indicated by <a href="#errors">the WebAssembly error mapping</a>.
     1. If |outArity| is 0, return undefined.
-    1. Otherwise, return [=ToJSValue=](|v|), where |v| is the singular element of |ret|.
+    1. Otherwise, if |outArity| is 1, return [=ToJSValue=](|v|), where |v| is the singular element of |ret|.
```

Comment:
nit: could we rename it to `retArity`?

Reply:
Feel free to propose that in a different PR (maybe against the main branch). outArity isn't a new name here.